account_bank_statement_import_camt_oca/models/account_journal.py

```diff
@@ -1,12 +1,16 @@
 # Copyright 2019 ACSONE SA/NV <thomas.binsfeld@acsone.eu>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import models, _
+from odoo import fields, models, _
 
 
 class AccountJournal(models.Model):
     _inherit = "account.journal"
 
+    group_return_statement_at_camt_import = fields.Boolean(
+        string="Group Return Statements At CAMT Import"
+    )
```

Comment:
Should we not name this flag "Ignore CAMT transaction details". help="If checked, import one statement line per CAMT entry else import on statement line per transaction detail".